dbos/core.py

```diff
@@ -289,10 +293,15 @@ def _execute_workflow_id(dbos: "DBOS", workflow_id: str) -> "WorkflowHandle[Any]
                 )
 
 
-def _workflow_wrapper(dbosreg: "_DBOSRegistry", func: F) -> F:
+def _workflow_wrapper(
+    dbosreg: "_DBOSRegistry",
+    func: F,
+    max_recovery_attempts: int = DEFAULT_MAX_RECOVERY_ATTEMPTS,
```

Comment:
I think it'll be nicer to make it a kwarg, because we require people to explicitly specify it.

Reply:
Okay saw it in dbos.py.